cmd/lncli/commands.go

```diff
@@ -1982,10 +1994,10 @@ var updateChannelPolicyCommand = cli.Command{
 	Action: actionDecorator(updateChannelPolicy),
 }
 
-func parseChanPoint(s string) (*lnrpc.ChannelPoint, error) {
+func parseChannelPointStr(s string) (*lnrpc.ChannelPoint, error) {
 	split := strings.Split(s, ":")
-	if len(split) != 2 {
-		return nil, fmt.Errorf("expecting chan_point to be in format of: " +
+	if len(split) != 2 || len(split[0]) == 0 || len(split[1]) == 0 {
+		return nil, errors.New("expecting channel point to be in format of: " +
```

Comment:
nit: define this error as a global variable so you can re-use it in the unit tests.

Reply:
good idea. done